jpa/eclipselink.jpa.test/src/org/eclipse/persistence/testing/models/jpa/advanced/MyTestEntity.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 1998, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1998, 2018 IBM Corporation. All rights reserved.
```

Comment:
This is apparently new file, created in 2018 by Oracle, no need to say 1998 nor IBM...